daemon-tests/src/lib.rs

```diff
@@ -361,10 +361,10 @@ impl Taker {
         }
     }
 
-    pub async fn trigger_rollover(&self, id: OrderId) {
+    pub async fn trigger_rollover(&self, order_id: OrderId, position: Position) {
```

Comment:
I thought for a while that this is pretty weird API to need to pass the position, but then realised it's just for the tests..... 😌

Reply:
To be fair it's still pretty weird to me, but I couldn't come up with anything better at this stage.

Reply:
I think this might go away if we return the position together with the event from the command handler?